fix: add documentation of UserInfo policy (#24)
gravitee-io/gravitee-policy-openid-connect-userinfo: README.adoc

|===

== Description

> You can use the `policy-openid-userinfo` to get the OpenId Connect user info from an OAuth2 resource through its UserInfo endpoint.

Reviewer: ```suggestion
Use the `policy-openid-userinfo` to get the OpenId Connect user info from an OAuth2 resource through its `UserInfo` endpoint.
```

> NOTE: If no Oauth2 resource is defined for the policy, or it is not well configured, the request will fail with a 401 status. More information will be available in `WWW_Authenticate` header

Reviewer 2: ```suggestion
NOTE: If no Oauth2 resource is defined for the policy, or misconfigured, the request will fail with a 401 status. More information will be available in `WWW_Authenticate` header
```

Reviewer: please see the other variant below

> NOTE: If no Oauth2 resource is defined for the policy, or it is not well configured, the request will fail with a 401 status. More information will be available in `WWW_Authenticate` header

Reviewer: ```suggestion
NOTE: The request will fail with a 401 status If the policy's Oauth2 resource is misconfigured or not defined at all. To troubleshoot this, check the `WWW_Authenticate` header for more information.
```

== Configuration

> You can configure the policy with the following options:

Reviewer: ```suggestion
Use the following options to configure the policy:
```

|===
|Property |Required |Description |Type |Default

|oauthResource|X|The OAuth2 resource used to get UserInfo|string|

> |extractPayload||When `true`, the payload of the response from UserInfo endpoint is set in the gateway `openid.userinfo.payload` attribute|boolean|

Reviewer: ```suggestion
|extractPayload||When set to `true`, the payload of the response from the UserInfo endpoint is set in the `openid.userinfo.payload` gateway attribute|boolean|
```
